dev/src/main/resources/META-INF/plugin.xml

```diff
@@ -14,8 +14,50 @@
     <vendor email="codewind-dev@eclipse.org" url="https://www.eclipse.org/codewind/">Codewind</vendor>
 
     <description><![CDATA[
-        Adds support for developing cloud-native, containerized web applications.
-    ]]>    </description>
+      <h2>Codewind</h2>
+      <p>
+        Adds tools to help build cloud-native, containerized Java
+        applications. Create an application from a template project
+        and launch, update, test, and debug in containers on the
+        desktop. Supports Codewind version 0.10.0. For more
+        information, see <a href="https://www.eclipse.org/codewind/">https://www.eclipse.org/codewind/</a>.
+      </p>
+      <h2>Why use Codewind?</h2>
+      <ul>
+        <li>Get started with templates or samples to create and deploy applications that run in containers.</li>
+        <li>Create containerized applications in languages and runtimes that you're already familiar with.</li>
+        <li>Pull in your existing applications and use Codewind to help get them cloud ready.</li>
+        <li>See code changes reflected in your containerized application.</li>
+      </ul>
+      <h2>Features</h2>
+      <ul>
+        <li>Create new containerized projects or add existing ones.</li>
+        <li>View your containerized projects, including the application and build status.</li>
+        <li>Access application, build, and container logs in the Output view.</li>
+        <li>Toggle the project auto build setting and manually initiate project builds.</li>
+        <li>Open your application in a browser.</li>
+      </ul>
+      <h2>Getting started with Codewind</h2>
+      <p>
+        For more information, see
+        <a href="https://www.eclipse.org/codewind/intellij-getting-started.html">https://www.eclipse.org/codewind/intellij-getting-started.html</a>.
```

Comment:
Hyperlink with title of the file (minus "Tech Preview"): `<a href="https://www.eclipse.org/codewind/intellij-getting-started.html">Installing Codewind for IntelliJ</a>`

Reply:
Changed